### ISSUE TITLE
Uncaught TypeError: Cannot read property 'focus' of null

### ISSUE THREAD

**Issue by balajiraot** [Availity/react-block-ui#5]
testing with version: 0.2.3

getting _`Uncaught TypeError: Cannot read property 'focus' of null`_ from 
https://github.com/Availity/react-block-ui/blob/master/src/BlockUi.js at line 66 
```javascript
 if(this.focused && this.focused !== document.body) {
            setImmediate(() => this.topFocus.focus()) // <--------here 
 }
```
Sample code to recreate the issue 
```javascript
import React from 'react'
import BlockUi from 'react-block-ui'

export default class TestComp extends React.Component {
  constructor(props){
    super(props)
    this.state = {text:'',progress:false}
  }

  render (){
    return (<div>
      <BlockUi tag="div" blocking={this.state.progress}>
        <label>This is Sample</label>
        <input type="text" value={this.state.value} onChange={(e)=>{
          this.setState({text:e.value,progress:true})
          Promise.resolve({})
            .then(()=>{
              this.setState({text:e.value,progress:false})
            })
        }}/>
      </BlockUi>
    </div>)
  }
}
```

**Comment by TheSharpieOne** [comment]
You are essentially blocking then immediately unblocking within the same event loop. This is an unlikely use-case as most of the time this will be used when I/O will happen, but it's easy enough to support... now just need to determine if I want to simply do a null check or clear the `setImmediate`... why not both 😄

**Comment by TheSharpieOne** [comment]
Fixed in 0.2.4
